packages/data-layer/src/data-layer.ts

```diff
@@ -474,29 +475,29 @@ export class DataLayer {
       chainId,
     };
 
-    const response: { rounds: RoundWithApplications[] } = await request(
+    const response: { rounds: RoundForExplorer[] } = await request(
       this.gsIndexerEndpoint,
-      getRoundByIdAndChainIdWithApprovedApplications,
+      getRoundForExplorer,
       requestVariables,
     );
 
     if (response.rounds.length === 0) {
       return null;
     }
 
-    const _round = response.rounds[0] ?? [];
+    const round = response.rounds[0] ?? [];
```

Comment:
not related to this PR in particular but having `[]` by default is wrong because we are callung `round.applications` after

Reply:
ah yes i missed that, fixed